model.py

```diff
@@ -8988,7 +9005,7 @@ class Collection(Base):
     # to Overdrive Advantage collections.
     children = relationship(
         "Collection", backref=backref("parent", remote_side = [id]),
-        uselist=False
+        uselist=True
```

Comment:
Unrelated bug which showed up once I actually looked at `Collection.children` in a test.